docs/review-tool/README.md

```diff
@@ -40,22 +28,13 @@ There are five main steps that compose this process to build up our library:
 With these steps we have a feedback loop that leverages a new flow of information and raises the standards
 of the converted files.
 
-The Review Tool is an important piece of this puzzle and now our community can join forces with us and help
-to build up this library of agreements.
-
-# How-To
-
-## Sign Up
-
-- [Register](https://conversion.openlaw.io/register)
-
-## Sign In
-
-- [Login](https://conversion.openlaw.io/login)
+The Review Tool is an important piece of this puzzle and now our community can [sign up](https://conversion.openlaw.io/register)
```

Comment:
I don't know if it's a good idea to call it a puzzle, it could give the impression of something being confusing, maybe ecosystem?